bdd_tests/features/steps/common.py

```diff
@@ -0,0 +1,40 @@
+# Copyright © 2021 Pavel Tisnovsky, Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from shutil import which
+import psycopg2
+from psycopg2.errors import UndefinedTable
+
+
+from behave import given, then, when
+
+
+@given(u"the system is in default state")
+def system_in_default_state(context):
```

Comment:
Is this function's logic expected? It does nothing

Reply:
It's helpful in scenarios where we don't need a specific setup. With this, we can start them with "Given the system is in default state"

Reply:
oh ok, it's like a placeholder